src/extensions/mvs/tree/mvs/mvs-tree-primitives.ts

```diff
@@ -102,14 +135,85 @@ const PrimitiveLabelParams = {
     label_offset: OptionalField(float, 0, 'Camera-facing offset to prevent overlap with geometry.'),
 };
 
+const EllipsisParams = {
+    /** Color of the primitive. If not specified, uses the parent primitives group `color`. */
+    color: OptionalField(nullable(ColorT), null, 'Color of the ellipsis. If not specified, uses the parent primitives group `color`.'),
+    /** If true, ignores radius_minor/magnitude of the minor axis */
+    as_circle: OptionalField(bool, false, 'If true, ignores radius_minor/magnitude of the minor axis.'),
```

Comment:
We could get rid of this param and instead implement "if there is not value for minor radius, use the same as major radius".

Reply:
I am not sure this would work due to the presence of both "automated" radius from the axes and the manual radius. And both axes need to be provided to determine the plane of the ellipsis.